Jenkinsfile

```diff
@@ -7,5 +7,20 @@ node {
   govuk.buildProject(
     publishingE2ETests: true,
     brakeman: true,
+    extraParameters: [
+      stringParam(
+        name: "GDS_API_PACT_BRANCH",
+        defaultValue: "master",
```

Comment:
It's unfortunate we need to hard code this! I'm going to make a note of it: https://trello.com/c/zcuPB4dq/212-update-master-references-in-pact-tests